style.css

```diff
@@ -1,8 +1,8 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
```

Comment:
- [ ] You have done a good job with your mobile menu :muscle: but there is an style issue because your design shows the menu and the (X) icon above the mobile menu, and according to the Figma design, the mobile menu should be on top of all the content (including the toolbar) and the (X) should be inside of the mobile menu. I would suggest updating this issue in order to match the template :smiley: :raised_hands:
- [ ] Also, I think it could be a good idea to add some blur to the content behind the mobile menu because currently, the transparency can make the menu confusing. Please, kindly address this issue :wink:

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/146048200-9aeede5d-56fb-4f1d-a69d-6075268ff883.png) | ![image](https://user-images.githubusercontent.com/81522435/146047861-65e4a3ff-23d4-4027-959c-76c3ac41cd90.png) |

Reply:
thanks, @alonsofl for your feedback, I was trying to make the menu as the same as the template but I cannot make it like that because of the structure of my code so in order to make it this will means to do the entire header section :(